Project.toml

```diff
@@ -0,0 +1,18 @@
+name = "UnitfulAstro"
+uuid = "f2a1aa4e-5634-4978-a6f6-8c9e368f14ef"
```

Comment:
How did you get this UUID?  It's wrong

Reply:
Ah, I had just randomly generated a UUID, I did not think to check for the UUID from the julia registry

Reply:
```julia
julia> using Pkg

julia> Pkg.METADATA_compatible_uuid("UnitfulAstro")
UUID("6112ee07-acf9-5e0f-b108-d242c714bf9f")
```
:wink: